sdk/react/index.android.js

```diff
@@ -110,11 +110,8 @@ var OoyalaSkin = React.createClass({
       case SCREEN_TYPES.END_SCREEN:   
         return OoyalaSkinCoreInstance.renderEndScreen();   
         break;
-      case SCREEN_TYPES.MOREOPTION_SCREEN:
-        return OoyalaSkinCoreInstance._renderMoreOptionScreen();  
```

Comment:
Line 107 handles this case already and line 114 will never be executed. Also there's no method with name **_render**MoreOptionScreen